packages/flame_forge2d/lib/forge2d_game.dart

```diff
@@ -54,15 +53,6 @@ class Forge2DGame extends BaseGame {
     }
   }
 
-  @override
-  void remove(Component component) {
```

Comment:
game has no "remove" no more :) 
also this would not work for removeAll before.